app/p1-monitor/build.sbt

```diff
@@ -18,9 +18,9 @@ ocsAppManifest := {
       common(ov, pv),
         dev(ov, pv),
         staging(ov, pv),
-        v2015B(ov, pv),
         v2016B(ov, pv),
-        v2016B_2(ov, pv)
+        v2016B_2(ov, pv),
+	v2017A(ov, pv)
```

Comment:
just to be pedantic - why v2017A isn't aligned with the others?.... other than that.... 👍  ship it